### DISCUSSION TITLE
Save and load setup without saving to file?

### DISCUSSION THREAD

**Discussion by jmrichardson** [pycaret/pycaret#1992] [Q&A]
Hi, after doing setup and saving to a variable (inside a class method), I would like to predict in another class method and need to reload the setup variable.  I know I can do it via the save and load but don't want to write to a file.  Is it possible to load the global variables using just the setup variable?

**Comment by moezali1** [comment]
@jmrichardson You can save and load global variables using the `save_config` and `load_config` function but that's also a pickle file at the end of the day. Also, not a highly recommended way to retrieve the model but theoretically all models are part of a list that is a global variable so if you want you can use it that way. 

May I know more about the use-case of inference without file?
